### PR TITLE
Add aten::_foreach_{max, rsqrt, sqrt_, lerp.ScalarList, lerp_.ScalarList}

### PR DESCRIPTION
**Pull request by LuFinch**
- _foreach_max
- _foreach_rsqrt
- \_foreach_rsqrt_
- _foreach_lerp.ScalarList
- \_foreach_lerp_.ScalarList

### CODE REVIEW

File: src/ATen/native/xpu/sycl/ForeachTernaryKernels.cpp [hunk 1]
```diff
@@ -123,4 +123,58 @@ void foreach_lerp_scalar_kernel_(
             weight.to<opmath_t>());
       });
 }
+
```
Comment (xytintel) [inline src/ATen/native/xpu/sycl/ForeachTernaryKernels.cpp:126]: Please include `src/ATen/native/xpu/sycl/ForeachTernaryOpScalarListKernels.h`

File: src/ATen/native/xpu/sycl/ForeachReduceKernels.cpp [hunk 2]
```diff
@@ -476,4 +477,274 @@ std::vector<Tensor> foreach_norm_kernel(
...
+
+    if (n % kILP == 0 && (chunk_size & kILP) == 0 && is_aligned(x)) {
+      for (int64_t i_start = item_id;
+           i_start * kILP < n && i_start * kILP < chunk_size;
+           i_start += local_range) {
+        // load
+        load_store(r_x, x, 0, i_start);
+#pragma unroll
+        for (int ii = 0; ii < kILP; ii++) {
+          vals[ii] = max_impl(vals[ii], r_x[ii]);
```
Comment (xytintel) [inline src/ATen/native/xpu/sycl/ForeachReduceKernels.cpp:517]: Please use `max_propagate_nan`

**Comment by xytintel** [comment]
- foreach_lerp
image

**Comment by xytintel** [comment]
- foreach_rsqrt
image

**Comment by xytintel** [comment]
- foreach_max
 image

**Review by xytintel — APPROVED** [review]

**Comment by LuFinch** [comment]
@xytintel  Thanks for your review and help on solving comments. Now it seems that CI has issue on inductor E2E conda env building.

**Pull request merged**